react/components/BuyButton/index.js

```diff
@@ -2,6 +2,7 @@ import find from 'lodash/find'
 import PropTypes from 'prop-types'
 import React, { Component, Fragment } from 'react'
 import { injectIntl, intlShape, FormattedMessage } from 'react-intl'
+import ContentLoader from 'react-content-loader'
 
 import { contextPropTypes, orderFormConsumer } from 'vtex.store/OrderFormContext'
 import { Button } from 'vtex.styleguide'
```

Comment:
Add the new variable to the `state` property of this class, setting a default. And also, on line 52 there's a setState to loading. Is this correct?